db/warehouse/migrate/20250513134455_create_fy2026_loading_and_import_tables.rb

```diff
@@ -0,0 +1,385 @@
+# frozen_string_literal: true
+
+class CreateFy2026LoadingAndImportTables < ActiveRecord::Migration[7.1]
+  def change
+    StrongMigrations.disable_check(:add_index) # Indexes are created inside hmis_table_create_indices!, so don't complain about them
+
+    spec_version = '2026'
+
+    # Loader tables
+    HmisCsvTwentyTwentySix.loadable_files.each_value do |klass|
+      klass.hmis_table_create!(version: spec_version, constraints: false, types: false)
+      klass.hmis_table_create_indices!(version: spec_version, ignored_indexes: ignored_loader_indexes)
+    end
+
+    HmisCsvTwentyTwentySix.loadable_files.each_value do |klass|
+      column_names = klass.column_names
+      add_column klass.table_name, :data_source_id, :integer, null: false
```

Comment:
I think we are trying to get away from using integers for pks. 
```suggestion
      add_reference klass.table_name, :data_source, null: false # reference should be a bigint
```